sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/ContainersOperations.cs

```diff
@@ -91,7 +95,7 @@ internal ContainersOperations(ContainerInstanceManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), bool? timestamps = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This seems like a breaking change: introducing new variables not at the end.

Please rearrange `bool? timestamps = default(bool?)` in the swagger file so it's at the end.